src/Analyzers/CSharp/Tests/ConvertSwitchStatementToExpression/ConvertSwitchStatementToExpressionTests.cs

```diff
@@ -15,6 +13,14 @@
 using Roslyn.Test.Utilities;
 using Xunit;
 
+#if CODE_STYLE
```

Comment:
These #if statements are needed in shared test project for unit tests that need any CodeStyleOption or CSharp/VisualBasic CodeStyleOption. These are defined in Workspaces layer as well as linked into CodeStyleLayer so that CodeStyle analyzers can use them in a Workspace-agnostic way. I have switched the linked implementations in CodeStyleLayer to use a separate namespace (`.Internal`) to avoid need of extern aliases, which are not available in VB and make the code more difficult to understand for C#.